src/cpp/session/modules/SessionProfiler.R

```diff
@@ -13,4 +13,54 @@
 #
 #
 
+.rs.addJsonRpcHandler("start_profiling", function(profilerOptions)
+{
+   tryCatch({
+      tempPath <- .Call("rs_profilesPath")
+      if (!dir.exists(tempPath)) {
```

Comment:
K, changing.

BTW Is there a way to find incompatible R code with our version? I guess one option would be to switch to an older R version, correct? What would you recommend for finding these issues out?